src/plopp/backends/common.py

```diff
@@ -8,17 +8,22 @@
 
 import numpy as np
 import scipp as sc
+from matplotlib import dates as mdates
```

Comment:
This is in the `common` backend module. Shouldn't it be in the matplotlib module?

Reply:
Hmm, good point